app/javascript/src/components/OnboardingFollowUsers.jsx

```diff
@@ -10,4 +17,11 @@ const OnboardingFollowUsers = ({ users, checkedUsers, handleCheckUser, handleChe
   />
 );
 
+OnboardingFollowUsers.propTypes = {
+  users: PropTypes.arrayOf(userPropType).isRequired,
+  checkedUsers: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
It's the same as using the type `any`. I think in these cases we should use `PropTypes.shape` and define the type of each property.

Reply:
I wasn't sure what to put there so i just left it at object. I'll try to investigate and see what checkedUsers contains...

Reply:
Yes, in this case it's really hard to tell (maybe in a runtime debugger tool). Normally, when I'm not sure about the entire shape, I just define those props that are used by the component. But in that case, it's just passed through.

Since more than one component accept this object as a property, this prop type should have it's own file and should be re-usable in order to avoid re-defining the entire shape all over again.

Reply:
@emimuresan not very orthodox but you can just console.log the property and manually check what "shape" it has.

Reply:
Couldn't get to the page it was used, but based on the code, `checkedUsers` contains user objects. So i used `userPropType` in the prop type description.